backends/sycl-ref/ceed-sycl-ref-operator.sycl.cpp

```diff
@@ -1251,35 +1280,35 @@ static int CeedOperatorLinearAssembleFallback_Sycl(sycl::queue &sycl_queue, cons
   // This kernel assumes B_in and B_out have the same number of quadrature points and basis points.
```

Comment:
Note - we merged a lot of dead code here in the SYCL backend. I don't love that.